.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ on:
         description: Enables GDB and Amalgam Fast Memory Integrity checks for all Linux/x86 Pytest jobs.
```

Comment:
Looks like my [prettier](https://prettier.io/) plugin caught a ton of lines here. The only real changes here are adding:

```yaml
  # enable-ruff: true
  # enable-pyright: true 
```